content/en/docs/concepts/services-networking/service.md

```diff
@@ -430,8 +430,7 @@ specifying `"None"` for the cluster IP (`.spec.clusterIP`).
 
 You can use a headless Service to interface with other service discovery mechanisms,
 without being tied to Kubernetes' implementation. For example, you could implement
-a custom [Operator](
-be built upon this API.
+a custom Operator upon this API.
```

Comment:
@sftim, the change is okay, though, the semantics seem a bit off to me. Can you clarify what `upon this API` vs `build an Operator to interface with [what API]?

Reply:
Maybe it's easier to remove the example? It looks like I saved some unfinished work here, and _I_ don't remember what point I wanted to make.

Reply:
Here is the current version:
`You can use a headless Service to interface with other service discovery mechanisms, without being tied to Kubernetes’ implementation. For example, you could implement a custom [Operator]( be built upon this API.`

- I _think_ the original idea is suggesting that the custom Operator handles: load balancing, proxying, DNS configuration for this service, but I may have misinterpreted the concept.

Reply:
I re-rewrote the text to clarify a bit.

If that's not helping, what do people think about chopping out the example instead?